substrate/frame/paged-list/src/mock.rs

```diff
@@ -62,7 +61,7 @@ pub type MetaOf<T, I> =
 	StoragePagedListMeta<ListPrefix<T, I>, <T as Config>::Value, <T as Config>::ValuesPerNewPage>;
 
 /// Build genesis storage according to the mock runtime.
-pub fn new_test_ext() -> sp_io::TestExternalities {
+pub fn new_test_ext() -> TestExternalities {
```

Comment:
It seems to be renamed into this.

```suggestion
pub fn new_test_ext() -> TestState {
```